packages/containers/psammead-timestamp-container/src/index.stories.jsx

```diff
@@ -13,6 +13,7 @@ storiesOf('Containers|TimestampContainer', module)
       format="D MMMM YYYY"
       isRelative={boolean('isRelative', false)}
       script={latin}
+      locale="ig"
```

Comment:
Can you use a locale like `fa`, and add a new story please. Would be good to have examples of different locales in storybook